modules/multi-architecture-enabling-64k-pages.adoc

```diff
@@ -0,0 +1,89 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/multi-architecture-configuration.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="multi-architecture-enabling-64k-pages_{context}"]
+
+= Enabling 64k pages on the {op-system-first} kernel 
+
+You can enable 64k memory page in the {op-system-first} kernel on the 64-bit ARM compute machines in your cluster. The 64k page size kernel specification can be used for large GPU or high memory workloads. This is done using the Machine Config Operator (MCO) which uses a machine config pool to update the kernel. To enable 64k page sizes, you must dedicate a machine config pool for ARM64 to enable on the kernel.
+
+[IMPORTANT]
+====
+Using 64k pages is exclusive to 64-bit ARM architecture compute nodes or clusters installed on 64-bit ARM machines. If you configure the 64k pages kernel on a machine config pool using 64-bit x86 machines, the machine config pool and operator will degrade.
+====
+
+.Prerequisites: 
+* You installed the OpenShift CLI `oc`
+* You created a cluster with compute nodes of different architecture on one of the supported platforms
+
+.Procedure: 
+
+. Label the nodes where you want to run the 64k page size kernel:
+[source,terminal]
++
+----
+$ oc label node <node_name> <label>
+----
++
+[source,terminal]
+----
+$ oc label node worker-arm64-01 node-role.kubernetes.io/worker-64k-pages=
+----
+
```

Comment:
Actually, I don't think we need both a generic command and a command with example values. I might go with the latter, so that the user can see the label being used in the MCP.

Reply:
I believe QE wanted to show the generic command and we thought having an example would just be helpful for users.